applications/launchpad/gui-react/src/components/Onboarding/OnboardingMessages/index.tsx

```diff
@@ -0,0 +1,31 @@
+import Text from '../../Text'
+import t from '../../../locales'
+
+const Message1 = (
```

Comment:
if these are indeed just messages that appear in order, it would be better to have an array of messages instead of individual Message1, Message2 values, like:
`const messages = [(...), (...), (...)]`
because then you can access them like
`messages[0]` - easier to iterate and it doesn't look like lazy naming :D

(just a suggestion, I don't like the Message1, Message2 names etc)

Reply:
Done!